backend/apps/matching/models/participant_info_location.py

```diff
@@ -7,7 +7,7 @@
 import numpy as np
 
 from apps.matching.files.map_data import plzs
-from apps.matching.src.plzs import BIG_CITY_PLZS
+from apps.matching.utils.plzs import BIG_CITY_PLZS
```

Comment:
as we're cleaning this up, suggest renaming to zipcode? Should we use singular or plural here?